core/client/vendor/theme-ghost.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * An Ace (http://ace.c9.io) Code Editor Theme for Ghost
+ */
+/* global ace */
+
+/* jshint multistr: true */
+ace.define('ace/theme/ghost', ['require', 'exports', 'module', 'ace/lib/dom'], function (require, exports) {
+    
+    exports.isDark = false;
+    exports.cssClass = 'ace-ghost';
+    exports.cssText = '\
```

Comment:
I believe this'll get es6 transformed if you want to use backticks for literal strings, ie

```js
`
.ace-ghost .ace_gutter {
  background: foo,
  bar: baz
}
`
```